### ISSUE TITLE
WSL 2 issues with libnss3.so

### ISSUE THREAD

**Issue by RodrigoCMoraes** [wdsrocha/dojo#22]
While I was starting [server](https://github.com/wdsrocha/dojo/tree/main/packages/server) I got issues with `libnss3.so` and something with chromedriver, but I didn't save it :/ 

Installing `apt install libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev` I was able to solve the issue, [reference](https://stackoverflow.com/a/65107381/7195250).

**Comment by RodrigoCMoraes** [comment]
Now I got the error o/

```shell
 yarn start:dev
yarn run v1.22.11
$ nest start --watch
[9:32:00 PM] Starting compilation in watch mode...

[9:32:04 PM] Found 0 errors. Watching for file changes.
[9:32:06 PM] File change detected. Starting incremental compilation...

[Nest] 2214   - 08/21/2021, 9:32:06 PM   [NestFactory] Starting Nest application...
[Nest] 2214   - 08/21/2021, 9:32:06 PM   [InstanceLoader] PassportModule dependencies initialized +27ms
[Nest] 2214   - 08/21/2021, 9:32:06 PM   [InstanceLoader] QueueModule dependencies initialized +0ms
[Nest] 2214   - 08/21/2021, 9:32:06 PM   [InstanceLoader] DatabaseModule dependencies initialized +0ms
[Nest] 2214   - 08/21/2021, 9:32:06 PM   [InstanceLoader] TypeOrmModule dependencies initialized +0ms
[Nest] 2214   - 08/21/2021, 9:32:06 PM   [InstanceLoader] UriModule dependencies initialized +1ms
[Nest] 2214   - 08/21/2021, 9:32:06 PM   [InstanceLoader] PuppeteerModule dependencies initialized +0ms
[Nest] 2214   - 08/21/2021, 9:32:06 PM   [InstanceLoader] ConfigHostModule dependencies initialized +0ms
[Nest] 2214   - 08/21/2021, 9:32:06 PM   [InstanceLoader] DiscoveryModule dependencies initialized +0ms
[Nest] 2214   - 08/21/2021, 9:32:06 PM   [InstanceLoader] ConfigModule dependencies initialized +1ms
[Nest] 2214   - 08/21/2021, 9:32:06 PM   [InstanceLoader] ConfigModule dependencies initialized +0ms
[Nest] 2214   - 08/21/2021, 9:32:06 PM   [InstanceLoader] BullModule dependencies initialized +0ms
[Nest] 2214   - 08/21/2021, 9:32:06 PM   [InstanceLoader] AppModule dependencies initialized +1ms
[Nest] 2214   - 08/21/2021, 9:32:06 PM   [InstanceLoader] BullModule dependencies initialized +18ms
[Nest] 2214   - 08/21/2021, 9:32:06 PM   [InstanceLoader] JwtModule dependencies initialized +0ms
[Nest] 2214   - 08/21/2021, 9:32:06 PM   [InstanceLoader] BullModule dependencies initialized +3ms
[Nest] 2214   - 08/21/2021, 9:32:06 PM   [ExceptionHandler] Failed to launch the browser process!
/home/rodrigocmoraes/code/dojo/node_modules/puppeteer/.local-chromium/linux-818858/chrome-linux/chrome: error while loading shared libraries: libnss3.so: cannot open shared object file: No such file or directory


TROUBLESHOOTING: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md
 +17ms
Error: Failed to launch the browser process!
/home/rodrigocmoraes/code/dojo/node_modules/puppeteer/.local-chromium/linux-818858/chrome-linux/chrome: error while loading shared libraries: libnss3.so: cannot open shared object file: No such file or directory


TROUBLESHOOTING: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md

    at onClose (/home/rodrigocmoraes/code/dojo/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserRunner.js:193:20)
    at Interface.<anonymous> (/home/rodrigocmoraes/code/dojo/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserRunner.js:183:68)
    at Interface.emit (events.js:326:22)
    at Interface.close (readline.js:416:8)
    at Socket.onend (readline.js:194:10)
    at Socket.emit (events.js:326:22)
    at endReadableNT (_stream_readable.js:1241:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
^C
```

**Comment by wdsrocha** [comment]
Fixed in #24
